File: write_movie_file.py
import os
import moviepy.video.io.ImageSequenceClip
image_folder='/home/d9smith/projects/age/volume/plots/test_images'
fps=1

image_files = [os.path.join(image_folder,img)
               for img in os.listdir(image_folder)
               if img.endswith(".png")]
clip = moviepy.video.io.ImageSequenceClip.ImageSequenceClip(image_files, fps=fps)
clip.write_videofile('/home/d9smith/projects/age/volume/my_test_video.mp4')

File: write_movie_file.m
% combine still images into movie

% If your image file starts with image_1.png, image_2.png and so on ...
% and live in the current folder.
folder = pwd; % Or wherever you want.
video = VideoWriter('testvideo.mp4'); % Create the video object.
video.FrameRate = 1;
open(video); % Open the file for writing
N=8; % Where N is the separate number of PNG image files.
for k = 1 : N 
    I = imread(fullfile(folder, 'volume', 'plots', sprintf('area_ic5_sm1000_designmat1_SAgeSexScanSoft_age_%g.0.png', k+8))); % Read the next image from disk.
    writeVideo(video,I); % Write the image to file.
end
close(video); 

% /home/d9smith/projects/age/volume/plots/area_ic5_sm1000_designmat1_SAgeSexScanSoft_age_9.0.png